src/test/java/APICalls/PostCall.java

```diff
@@ -0,0 +1,38 @@
+package APICalls;
+
+import JacksonModels.Response;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+
+
+import java.io.*;
+import java.util.Properties;
+
+import static utility.PropertyManager.getProperties;
+
+public class PostCall extends BaseCall {
+
+    protected static String POST_URL = "https://api.trello.com/1/boards/?name=test_board&key="+ apiKey+"&token="+ apiToken;
```

Comment:
URL and body for the POST request are statically defined, which reduces flexibility
consider making these dynamic to support different kinds of POST requests
same for GET